components/pages/game/components/GameProvider.tsx

```diff
@@ -18,8 +18,8 @@ const GameProvider = ({
   const [startGame, startGameState] = useStartGame();
 
   useEffect(() => {
-    startGame();
-  }, [startGame]);
+    if (imagesState?.images) startGame();
```

Comment:
This line fifes the problem, but maybe this problem will disappear soon, when we remove the `useGameImages`